fabric-extensibility-api-v1/src/main/resources/fabric-extensibility-api-v1.accesswidener

```diff
@@ -0,0 +1,4 @@
+accessWidener	v1	named
+
+accessible    class    net/minecraft/entity/mob/DrownedEntity$TridentAttackGoal
+accessible    field    net/minecraft/entity/projectile/TridentEntity    tridentStack    Lnet/minecraft/item/ItemStack;
```

Comment:
Why an AW and not an accessor?

Reply:
I can fix that. Ive never really gotten used to accessors

Reply:
Swapped to an Accessor